client/reader/discover/discover-stream.js

```diff
@@ -91,6 +93,18 @@ const DiscoverStream = ( props ) => {
 		selectedStreamName: selectedTab,
 	};
 
+	if ( selectedTab === 'add-new' ) {
+		return (
+			<ReaderMain className={ clsx( 'following main', props.className ) }>
```

Comment:
edit - nevermind.  `following main` struck me as off here since this isn't the recent/following feed, but it looks like we use these consistently as base classes on all our streams.